src/SDK/Language/CLI.php

```diff
@@ -0,0 +1,156 @@
+                case self::TYPE_STRING:
+                    $output .= "'{$default}'";
+                    break;
+            }
+        }
+
+        return $output;
+    }
+
+    /**
+     * @return array
+     */
+    public function getFiles()
+    {
+        return [
+            [
+                'scope'         => 'default',
+                'destination'   => 'README.md',
+                'template'      => '/cli/README.md.twig',
+                'minify'        => false,
+                //'block'         => 'default',
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'CHANGELOG.md',
+                'template'      => '/cli/CHANGELOG.md.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'LICENSE',
+                'template'      => '/cli/LICENSE.md.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'composer.json',
+                'template'      => '/cli/composer.json.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'method',
+                'destination'   => 'docs/examples/{{service.name | caseLower}}/{{method.name | caseDash}}.md',
+                'template'      => '/cli/docs/example.md.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'src/{{ spec.title | caseUcfirst}}/Client.php',
+                'template'      => '/cli/src/Client.php.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'Dockerfile',
+                'template'      => '/cli/Dockerfile.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'src/{{ spec.title | caseUcfirst}}/Parser.php',
+                'template'      => '/cli/src/Parser.php.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'service',
+                'destination'   => '/bin/{{service.name}}',
+                'template'      => '/cli/bin/service.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'default',
+                'destination'   => '/bin/client',
+                'template'      => '/cli/bin/client.twig',
+                'minify'        => false,
+            ],
+            [
+                'scope'         => 'service',
+                'destination'   => '/src/{{ spec.title | caseUcfirst}}/Services/{{service.name | caseUcfirst}}.php',
```

Comment:
We usually locate functional/app files (non-oop / classes) in the `app` directory and not the `src`. We would also name then in all lowercase + dashes.

Reply:
Done